### ISSUE TITLE
Configuration reset to default after applying licenses and reboot

### ISSUE THREAD

**Issue by hameedjafar** [40net-cloud/fortinet-azure-solutions#8]
Hi,
I have successfully deployed the HA (Active/Passive) and connected over SSH and verified all the configurations.
Then, i connected over GUI and applied the licenses on both the Firewall.
Once done, i couldnt reach the firewalls either via GUI and SSH.
Then, i managed to login via Cluster IP (active node) thro SSH and noticed that, all the configuration of the active node were reseted to factory defaults with WAN interface on DHCP. 

Kindly, could you please shed some light on what went wrong.
Thanks
Hameed

**Comment by jvhoof** [comment]
Hi Hameed,
Have you been able to replicate this in a new deployment? I have not encountered this type of issue before. If you can replicate this I suggest contacting Fortinet support to get this reported as a bug.
Regards,
Joeri

**Comment by hameedjafar** [comment]
Hi Joeri,
Thanks for your response.
I had to delete the VMs, as i couldnt reach the Standby unit due to active cluster PIP only on Primary Firewall.
When i redeployed and applied the license, it rebooted and came back this time with all the configuration without Default routes. I had to SSH into it and applied the default route to get the license activated. The Standby unit also had the same issue and after applying the default routes, now both are up, but the HA is not syncing with all the correct configurations. The HA ports are pinging and the priorities, peer IPs are clean.

As you suggested, i will open a case with Fortinet and let them have a look.

Thanks very much
Hameed
